native/src/modules/settings/AppSettings.js

```diff
@@ -23,7 +25,9 @@ export const defaultSettings: SettingsType = {
   errorTracking: null,
   allowPushNotifications: null,
   proposeNearbyCities: null,
-  apiUrlOverride: null
+  apiUrlOverride: null,
```

Comment:
For all other settings we have specific getters and setters, but probably okay to just be able to set this with setSettings